pkg/pgclient/client.go

```diff
@@ -66,15 +66,25 @@ type Client struct {
 	metricCache   *pgmodel.MetricNameCache
 }
 
+//
```

Comment:
Better to add an explanation than leaving it empty to slient go-vet.

Reply:
Looks like a bad merge removed the comment...